docs/CHANGELOG.asciidoc

```diff
@@ -51,6 +52,222 @@ Review important information about the {kib} 8.x releases.
 * <<release-notes-8.0.0-alpha1>>
 
 --
+[[release-notes-8.11.0]]
+== {kib} 8.11.0
+
+For information about the {kib} 8.11.0 release, review the following information.
+
+[float]
+[[breaking-changes-8.11.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 8.11.0, review the breaking changes, then mitigate the impact to your application.
+
+[discrete]
+[[breaking-167085]]
+.Improve config output validation for default output.
+[%collapsible]
+====
+*Details* +
+Improve config output validation to not allow to defining multiple default outputs in {kib} configuration. For more information, refer to ({kibana-pull}167085[#167085]).
+====
+      
+[discrete]
+[[breaking-161806]]
+.Convert filterQuery to KQL.
+[%collapsible]
+====
+*Details* +
+Converts `filterQuery` to a KQL query string. For more information, refer to ({kibana-pull}161806[#161806]).
+====
+      
+[float]
+[[deprecations-8.11.0]]
+=== Deprecations
+
+The following functionality is deprecated in 8.11.0, and will be removed in 9.0.0.
+Deprecated functionality does not have an immediate impact on your application, but we strongly recommend
+you make the necessary updates after you upgrade to 8.11.0.
+
+[discrete]
+[[deprecation-164651]]
+.Updates to move from doc_root.vulnerability.package -> doc_root.package (ECS).
+[%collapsible]
+====
+*Details* +
+This updates all instances of vulnerability.package to the ECS standard package fieldset. For more information, refer to ({kibana-pull}164651[#164651]).
+====
+      
+[float]
+[[features-8.11.0]]
+=== Features
+{kib} 8.11.0 adds the following new and notable features.
+
+Alerting::
+* Elasticsearch query rule can select multiple group-by terms ({kibana-pull}166146[#166146]).
+* Adds a Log tab to the Observability Rules page ({kibana-pull}165115[#165115]).
+APM::
+* Adds bulk action to untrack selected alerts ({kibana-pull}167579[#167579]).
+* Introduce custom dashboards tab in service overview ({kibana-pull}166789[#166789]).
+* Adds service profiling Top 10 Functions ({kibana-pull}166226[#166226]).
+* Adds service profiling flamegraph ({kibana-pull}165360[#165360]).
+Cases::
+* Adds custom fields in Cases ({kibana-pull}167016[#167016]).
+Dashboard::
+* Copy panel refactor ({kibana-pull}166991[#166991]).
+* Make links panel available under technical preview ({kibana-pull}166896[#166896]).
```

Comment:
I'm not sure what to put here, but I would consider this a highlight. If we wanted a short blurb just for the release notes it could be something like:

```suggestion
* Adds a new "Links" panel in technical preview. The Links panel allows authors to link multiple Dashboard together with fast navigation that retains users' context. ({kibana-pull}166896[#166896]).
```